projects/fusion-ui/components/input/input.component.html

```diff
@@ -20,14 +20,23 @@
             </fusion-icon>
         </label>
 
-        <i class="icon {{ config?.iconPos }}"
+        <i class="fu-input-icon {{ config?.icon?.iconPos || config?.icon?.[0]?.iconPos }}"
            *ngIf="config?.icon && !loading && !config?.errorType"
            [class.has-units]="config?.units"
            [ngStyle]="config.options?.iconStyle">
-            <fusion-icon [name]="config?.icon"></fusion-icon>
+            <fusion-icon [name]="config?.icon?.iconData || config?.icon?.[0]?.iconData"></fusion-icon>
         </i>
 
-        <i class="icon loading {{ config?.iconPos }}"
+        <ng-container *ngIf="config?.icon?.[1]?.iconData">
+            <i class="fu-input-second-icon {{ config?.icon?.[1]?.iconPos }}"
+               *ngIf="config?.icon && !loading && !config?.errorType"
+               [class.has-units]="config?.units"
```

Comment:
please don't copy paste.
if you don't need  attributes then don't add them. for example:
```
[class.has-units]="config?.units"
*ngIf="config?.icon && !loading && !config?.errorType"
[ngStyle]="config.options?.iconStyle">
```